packages/vapor/resources/icons/svg/preview-error.svg

```diff
@@ -0,0 +1,26 @@
+<svg xmlns="http://www.w3.org/2000/svg" fill="none">
```

Comment:
The viewbox attribute is missing on this tag which causes the preview to be cropped and non-centered. The same thing will probably happen in the admin

![image](https://user-images.githubusercontent.com/260007/107780025-26a43000-6d14-11eb-85b2-70bd5cf7e887.png)